test/support/test_config.h

```diff
@@ -54,6 +54,8 @@
 #define _PSTL_STD_UNINITIALIZED_FILL_BROKEN (_MSC_VER == 1900)
 // GCC10 produces wrong answer calling exclusive_scan using vectorized polices
 #define TEST_GCC10_EXCLUSIVE_SCAN_BROKEN (_GLIBCXX_RELEASE == 10)
+// GCC7 not returning const rvalue reference from const rvalue reference of tuple
+#define TEST_GCC7_RVALUE_REFERENCE_FROM_TUPLE_BROKEN (_GLIBCXX_RELEASE && _GLIBCXX_RELEASE < 8)

```

Comment:
I would mention `get` method as well as a main root cause of the issue
```suggestion
// GCC7 not returning const rvalue reference from const rvalue reference overload of get
#define TEST_GCC7_RVALUE_TUPLE_GET_BROKEN (_GLIBCXX_RELEASE && _GLIBCXX_RELEASE < 8)

```

Reply:
Done